src/jdk.compiler/share/classes/com/sun/tools/javac/code/Lint.java

```diff
@@ -283,6 +286,11 @@ public String toString() {
          */
         STATIC("static"),
 
+        /**
+         * Warning about unnecessary uses of the strictfp modifier
```

Comment:
Could be better to align with other comments that use "Warn" as a verb.

Reply:
Good catch; fixed.